dashboard/app/controllers/course_offerings_controller.rb

```diff
@@ -37,7 +37,9 @@ def quick_assign_course_offerings
 
   def self_paced_pl_course_offerings
     return head :bad_request unless current_user
-    offerings = CourseOffering.assignable_course_offerings(current_user).select {|co| co.get_participant_audience == 'teacher' && co.instruction_type == 'self_paced'}
+    offerings = CourseOffering.assignable_course_offerings(current_user).filter do |co|
+      co.get_participant_audience == 'teacher' && co.any_version_is_in_published_state? && co.instruction_type == 'self_paced' && co.header.present?
```

Comment:
What is `co.header.present?` checking?

Reply:
It's the way we filter out course offerings that were created in LB but never meant to appear on curriculum quick assign (it represents a label 'self-paced' that these all have). For example, there are a couple sandbox course offerings and such that don't have headers. I believe the published state will also filter these out, but I wanted to be consistent with this marketing field from LB to match what we have in quick assign!

Reply:
Thank you gotcha!